src/functTest/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainerFunctionalTest.groovy

```diff
@@ -133,4 +134,94 @@ class DockerCreateContainerFunctionalTest extends AbstractFunctionalTest {
         result.output.contains("seven=eight")
         result.output.contains("nine=ten")
     }
+
+    def "Set autoRemove"() {
```

Comment:
Let's be more explicitly about the expected behavior in the title. Same for the title test case below.

Reply:
This was trying to follow convention :) I'll update them with a more descriptive spec form.

Reply:
The titles of existing test cases are not necessarily very expressive and should be changed as well. We'll clean up a bunch of existing code over the next couple of weeks.

Reply:
Titles have been changed on `master`.